server/modules/auth/repository/jwtRepository.js

```diff
@@ -0,0 +1,28 @@
+const db = require('../../../db/db')
+
+const blacklistToken = async (token, expiration, client = db) => {
+  client.none(`
```

Comment:
do not leave pending promises. alway add ```await```

Reply:
Ok